features/org.eclipse.equinox.executable.feature/README.md

```diff
@@ -1,9 +1,31 @@
 This feature contains the eclipse executables and includes the equinox launcher bundle and its fragments.
-It also host the native C source files to build the executable and launcher binaries for all supported platforms.
+It also hosts the native C source files to build the launcher executable and launcher library binaries for all supported platforms.
 
-## Building the native executable and launcher binaries
+The _Equinox launcher executable_ is the executable binary file that can be used launch an Equinox based applications like Eclipse and is for example named `eclipse.exe` (on Windows) or `eclipse` (on Linux and Mac).
+The _launcher library_ is the native dynamic/shared library (eclipse.dll, eclipse.so, ...) that contains the compiled native code called from the `JNIBridge` in [`org.eclipse.equinox.launcher`](../../bundles/org.eclipse.equinox.launcher).
 
-To just build the native binaries navigate to the `library/<window-system>` sub-directory of interest and run 
+# Building the native executable and launcher binaries
+
+## Required software
+
+The software listed below is required to build the Equinox native binaries on the current platform targeting the same OS and processor arch.
+
+- common
+    - a JDK 1.8 or later (more recent versions are usually not a problem)
+- Windows:
+    - Microsoft Visual C Compiler 2019 or later (earlier version maybe also work), the 'Visual Studio Community Edition' is sufficient.
+- Linux:
+    - GTK development files. The Package `libgtk-3-dev` is sufficient.
+- MacOS:
+    - XCode Command Line Tools (already present in XCode)
```

Comment:
I to build the Equinox binaries on Mac the requirements are the same as for SWT as we have discussed in https://github.com/eclipse-platform/eclipse.platform.swt/pull/1174#discussion_r1573204789.
@Phillipus or @HeikoKlare is that correct?

Reply:
@HannesWell Yes, I think so. I've built the binaries on my Mac with just the Command Line Tools installed.

Reply:
Thanks.